### PR TITLE
OSDOCS-7975: Added third install route to Nutanix docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-7975](https://issues.redhat.com/browse/OSDOCS-7975)

Version(s):
4.15 to 4.12

Link to docs preview:
[Installing a cluster on Nutanix](https://65599--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mmatsuta-rh** [comment]
LGTM

**Review by xenolinux — COMMENTED** [review]
I left one suggestion but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc [hunk 1]
```diff
@@ -7,11 +7,13 @@ include::_attributes/common-attributes.adoc[]
...
 
-* Using installer-provisioned infrastructure. Use the procedures in the following sections to use installer-provisioned infrastructure. Installer-provisioned infrastructure is ideal for installing in environments with air-gapped/restricted networks.
+* Using user-provisioned infrastructure. Complete the relevant steps outlined in the xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.adoc#installing-platform-agnostic[Installing a cluster on any platform] documentation. 
+
+* Using installer-provisioned infrastructure. Use the procedures in the following sections to use installer-provisioned infrastructure. Installer-provisioned infrastructure is ideal for installing in disconnected network environments.
```
[the remark this answers is not in the document]
Comment (abhatt-rh) [reply installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc:16]: Suggestion sounds good to me. Noticed that a few other section follow this pattern. For example: [Choosing a method to install OpenShift Container Platform on bare meta](https://65599--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/preparing-to-install-on-bare-metal#choosing-a-method-to-install-ocp-on-bare-metal) and 
[Choosing a method to install OpenShift Container Platform on IBM Power](https://65599--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/preparing-to-install-on-ibm-power#choosing-an-method-to-install-ocp-on-ibm-power)

**Review by abhatt-rh — COMMENTED** [review]
Peer review LGTM; left a comment for @xenolinux's suggestion

**Comment by kalexand-rh** [comment]
I've opened the `enterprise-4.15` branch. Please review your PR and update the applicable versions to also cover 4.15 if you expect that the changes in this PR will be valid in that branch.

File: installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc [hunk 2]
```diff
@@ -7,11 +7,13 @@ include::_attributes/common-attributes.adoc[]
...
 
-* Using installer-provisioned infrastructure. Use the procedures in the following sections to use installer-provisioned infrastructure. Installer-provisioned infrastructure is ideal for installing in environments with air-gapped/restricted networks.
+**Using the Assisted Installer**: The link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[{ai-full}] hosted at link:http://console.redhat.com[console.redhat.com]. The {ai-full} cannot be used in disconnected environments. When using the {ai-full} with the Prism Central API, you must create all of your own cluster resources. Installing with the {ai-full} also provides integration with Nutanix, enabling autoscaling. See xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc#installing-on-prem-assisted[Installing an on-premise cluster using the {ai-full}] for additional details.
```
Comment (racedo) [inline installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc:14]: I'd replace the "when using AI with the Prism Central API, you must create all your own resources" by the same explanation used in IPI: "The AI doesn't provision the underlying infrastructure for the cluster and users must provision the infrastructure before the running the installer" or something along these lines.

**Pull request merged**